user-manuals/import-app/inventory.md

```diff
@@ -0,0 +1,67 @@
+# Upload Inventory
+
+The `HotWax Commerce Import App` is useful for merchandisers needing to swiftly update inventory without manually editing CSV formats or navigating FTP file browsers for field mapping. This tool allows merchandisers to upload CSV files, save custom mappings directly, and identify errors before finalizing the upload.
+
+For example, when merchandisers manually filter warehouse inventory to allocate stock for other channels before full online sales, they can efficiently use the Import App. By uploading CSV files to the Order Management System, they can promptly reset inventory counts after receiving new stock. This functionality streamlines the process and minimizes the risk of errors in managing inventory data for merchandisers.
+
+## Step-by-Step Guide: Updating Inventory in Bulk through CSV File
+
+1. **Access the Import App:**
+   - Locate the HotWax Commerce Import app in the app launchpad. Log in with your user credentials. 
+
+   {% hint style="info" %}
+Only users with Super or Admin user permissions can log in to the Import App
+{% endhint %} 
+
+2. **Navigating to Inventory Page:**
+   - Within the Import app, the default page opened is the `Purchase Order` page. Navigate to the `Inventory` page from the left menu.
+
+3. **Uploading CSV File with Saved Mappings:**
+   - Click on the `upload` button and select the preferred CSV file.
+
+4. **Mapping Fields and Identifying Errors:**
+   - Map CSV file fields with corresponding HotWax Commerce fields.
+   - The following fields need to be mapped in HotWax Commerce:
+   
+     | Field               | Description                                                |
+     |---------------------|------------------------------------------------------------|
+     | Shopify Product SKU | The unique product identifier used by Shopify.          |
```

Comment:
we should write that this can be changed as the user might not always have SKU configured.